pop3/commands/uidl/uidl.go

```diff
@@ -25,10 +25,10 @@ func Process(c *Client, cmd Command) Reply {
 		}
 	}
 	envs := db.List(c.GetString("authenticated_as"))
-	if cmd.Args == "" {
-		msg := []string{"\r\n"}
+	if cmd.Args == "" { // clawsmail says this an invalid response
+		msg := []string{"unique-id listing follows"}
 		for i, env := range envs {
-			msg = append(msg, fmt.Sprintf("%v %v", i, env.ID))
+			msg = append(msg, fmt.Sprintf("%v %v", i, env.ID)) // shouldnt we count from 1 ?
```

Comment:
"The first message in the maildrop is assigned a message-number of
"1", the second is assigned "2", and so on, so that the nth message
in a maildrop is assigned a message-number of "n" from the rfc